install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -180,6 +180,8 @@ containers:
   - --statusPort
   - "{{ annotation .ObjectMeta `status.sidecar.istio.io/port` .Values.global.proxy.statusPort }}"
 {{- end }}
+  - --httpAdminEndpoint
```

Comment:
This needs to be renamed. Envoy's admin interface always uses HTTP regardless of whether is over TCP or UDS.